pkg/nsx/util/utils.go

```diff
@@ -663,3 +665,22 @@ func UpdateRequestURL(reqUrl *url.URL, nsxtHost string, thumbprint string) {
 		reqUrl.Path = strings.Join(urls, "/")
 	}
 }
+
+func MergeArraysWithoutDuplicate(old []string, new []string) []string {
```

Comment:
I think we had better use another name, as `new` is a keyword in Golang.

Reply:
Renamed to newArray, thanks!